package.json

```diff
@@ -8,22 +8,23 @@
     "cdk": "cdk"
   },
   "dependencies": {
-    "@aws-cdk/aws-codebuild": "^1.37.0",
-    "@aws-cdk/aws-codecommit": "^1.37.0",
-    "@aws-cdk/aws-codepipeline": "^1.37.0",
-    "@aws-cdk/aws-codepipeline-actions": "^1.37.0",
-    "@aws-cdk/aws-ecs": "^1.37.0",
-    "@aws-cdk/core": "^1.37.0",
-    "@cdk-cosmos/cdk-credentials-plugin": "^0.4.3",
-    "@cdk-cosmos/core": "^0.4.3",
-    "@cosmos-building-blocks/pipeline": "^0.4.3",
+    "@aws-cdk/aws-codebuild": "^1.47.0",
```

Comment:
1.47.0..? cosmos libs are pinned to 1.37.0. Will it cause any issues..? Should we also change in cosmos lib to 1.47.0..?

Reply:
nah, in libs you want to support the lowest posable, but as a consumer you want the latest (its a bit of a perspective thing)